experimental/DoubleAndHyperComplexes/src/Morphisms/simplified_complexes.jl

```diff
@@ -477,8 +477,7 @@ degree: 0  1
 ```
 """
 function minimize(c::FreeResolution)
-  ### to be repaired by Matthias Zach
-  # @assert is_graded(c[-1]) "complex does not consist of graded modules"
+  @assert is_graded(c[-1]) || is_local(base_ring(c[-1])) "complex does not consist of graded modules"
```

Comment:
Please include local case in assert message.

Reply:
I mean in the error message "complex does not consist of graded modules". To not hold up this PR, we can also do that in the next PR on free resolutions.

Reply:
What about "minimization only available in graded and local cases"

Reply:
@wdecker I do not understand. The error message currently is "complex does not consist of graded modules or modules over a local ring". How does that not address your concern?

Reply:
That is right and o.k. When I made my comment, it looked different. So I guess this changed again with one of your commits.